pkg/local-storage/member/controller/volumegroup/manager.go

```diff
@@ -378,11 +395,53 @@ func (m *manager) processLocalVolumeGroup(lvgName string) error {
 	if len(lvg.Spec.Volumes) == 0 {
 		return m.deleteLocalVolumeGroup(lvg)
 	}
+
+	// check if need to update group's accessibility
+	volumes, err := m.GetLocalVolumeByLocalVolumeGroup(lvg)
+	if err != nil {
+		m.logger.WithError(err).Error("Fail to get LocalVolume by Group")
+		return err
+	}
+	volumeAccessNodes := map[string]int{}
+	for _, volume := range volumes {
+		if volume.Spec.Config != nil {
+			for _, replica := range volume.Spec.Config.Replicas {
+				volumeAccessNodes[replica.Hostname]++
+			}
+		}
+	}
+	if len(volumeAccessNodes) > len(lvg.Spec.Accessibility.Nodes) {
+		count := -1
+		ready := true
+		var nodes []string
+		// check all volumes' replica are ready
+		for nodeName, nodeReadyVolumeCount := range volumeAccessNodes {
+			if count == -1 {
+				count = nodeReadyVolumeCount
+			} else if count != nodeReadyVolumeCount {
+				ready = false
+				m.logger.WithFields(log.Fields{"lvg": lvg.Name, "volumeAccessNodes": volumeAccessNodes}).
+					Error("Fail to update LocalVolumeGroup's Accessibility Nodes, replicas are not all ready.")
+				break
+			}
+			nodes = append(nodes, nodeName)
+		}
+
+		if ready {
+			// update Group's accessibility nodes
+			nLvg := lvg.DeepCopy()
+			nLvg.SetAccessibilityNodes(nodes)
+			patch := client.MergeFrom(lvg)
+			if err = m.apiClient.Patch(context.TODO(), nLvg, patch); err != nil {
+				return err
+			}
+		}
+	}
+
 	return m.addLocalVolumeGroup(lvg)
```

Comment:
these codes can be combined into a single function `m.addLocalVolumeGroup` and update cache after found a new PVC(volume) in LocalVolumeGroup.